_episodes/01-intro.md

```diff
@@ -185,8 +184,8 @@ She has 1520 samples in all and now needs to:
 2.  Calculate statistics for each of the proteins separately
     using a program her supervisor wrote called `goostats`.
 3.  Write up results.
-    Her supervisor would really like her to do this by the end of the month
-    so that her paper can appear in an upcoming special issue of *Aquatic Goo Letters*.
+    
+Her supervisor would really like her to do this by the end of the month so that her paper can appear in an upcoming special issue of *Aquatic Goo Letters*.
```

Comment:
What does this whitespace change do? I can't see a rendering difference, so I can't see the point I'm afraid.

Reply:
In the original lesson, the "Her supervisor would really like for her..." was apart of the list of three things she needed to do. One the same line as "3. Write up results" which does not make sense. I think this should be separate because she needs to do all three thing by the end of the month not just writing up her results.